src/screens/DetailScreen.js

```diff
@@ -165,10 +165,19 @@ DetailScreen.navigationOptions = ({ navigation, navigationOptions }) => {
             accessibilityLabel="Teilen Taste"
             accessibilityHint="Inhalte auf der Seite teilen"
           >
-            <Icon
-              xml={share(colors.lightestText)}
-              style={headerRight ? styles.iconLeft : styles.iconRight}
-            />
+            {device.platform === 'ios' ? (
+              <Icon
+                name="ios-share"
+                size={26}
+                iconColor={colors.lightestText}
+                style={headerRight ? styles.iconLeft : styles.iconRight}
+              />
+            ) : (
+              <Icon
+                xml={colors.lightestText}
```

Comment:
error! this needs to stay `xml={share(colors.lightestText)}`. i will fix that.

Reply:
thanks!